src/library/Uno.Toolkit.Material/Uno.Toolkit.WinUI.Material.csproj

```diff
@@ -45,30 +45,16 @@
 		<XamlMergeInput Include="Styles\Controls\v2\**\*.xaml" MergeFile="mergedpages.winui.v2.xaml" />
 	</ItemGroup>
 
-	<Choose>
-		<When Condition="'$(TargetFramework)'=='net5.0-windows10.0.18362'">
 
-			<ItemGroup>
-				<PackageReference Include="Microsoft.WindowsAppSDK" />
+	<ItemGroup Condition="'$(TargetFramework)'=='net5.0-windows10.0.18362'">
```

Comment:
@kazo0 we should avoid direct comparison to specific tfms - use something like "contains('windows')" to test for windows